ShelterModule/Models/Pets/PetResponse.cs

```diff
@@ -1,17 +1,30 @@
-﻿using ShelterModule.Models.Shelters;
-using System.ComponentModel.DataAnnotations;
+﻿using System.ComponentModel.DataAnnotations;
 
-namespace ShelterModule.Models.Pets
+namespace ShelterModule.Models.Pets;
+
+public sealed class PetResponse
 {
-    public sealed class PetResponse
-    {
-        public required Guid Id { get; init; }
-        public required Guid ShelterId { get; init; }
-        public required string Name { get; init; }
-        public required string Species { get; init; }
-        public required string Breed { get; init; }
-        public required DateTime Birthday { get; init; }
-        public required string Description { get; init; }
-        public required string Photo { get; init; }
-    }
+    [Required]
+    public required Guid Id { get; init; }
+
+    [Required]
+    public required Guid ShelterId { get; init; }
+
+    [Required]
+    public required string Name { get; init; }
+
```

Comment:
I'm 90% sure that Swagger does not see the `required` keyword, so we should have attributes there to signify that the response object will always contain those properties. 

In general, most reflection-based mechanisms work with attributes and don't check for the `required` keyword. EF Core is an exception from this.